src/config/reader.rs

```diff
@@ -348,15 +358,33 @@ mod test_proto_config {
         proto_no_pkg.pop();
         proto_no_pkg.push("proto");
         proto_no_pkg.push("news_no_pkg.proto");
-        let err = &validation.as_vec().first().unwrap().message;
+
+        if cfg!(windows) {
+            let err = &validation
+                .as_vec()
+                .first()
+                .unwrap()
+                .message
+                .replace('/', "\\\\");
```

Comment:
what's the logic behind this?

Reply:
err contained a / in file paths, so replaced it with \\ for windows. I wasn't able to find the root cause of why / was present in the first place.